cla_public/apps/base/views.py

```diff
@@ -234,3 +240,20 @@ def smoke_tests():
     from cla_public.apps.checker.tests.smoketests import SmokeTests
 
     return jsonify(smoketest(SmokeTests))
+
+
+@base.route('/ping.json')
+def ping():
+    return jsonify({
+        'version_number': os.environ.get('APPVERSION'),
+        'build_date': os.environ.get('APP_BUILD_DATE'),
+        'commit_id': os.environ.get('APP_GIT_COMMIT'),
+        'build_tag': os.environ.get('APP_BUILD_TAG')
+    })
+
+
+@base.route('/healthcheck.json')
+def healthcheck():
+    backend_healthcheck_uri = '%s/%s' % (current_app.config['BACKEND_BASE_URI'], 'status/healthcheck.json')
+    response = requests.get(backend_healthcheck_uri)
```

Comment:
should just test the response code is OK and return its own healthcheck rather than the backend one